src/boot.py

```diff
@@ -82,7 +82,7 @@ def home(ctx_home):
     """Loads and run the Login page"""
     from krux.pages.home_pages.home import Home
 
-    if ctx_home.wallet is not None:
+    if ctx_home.wallet is not None and ctx_home.wallet.key:
```

Comment:
This new form to check if a wallet is loaded would be better as a helper function in the context class

Reply:
Context method ".is_logged_in()" is the name I have in mind for this.